### DISCUSSION TITLE
Senta que lá vem a História #1

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#515] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/senta-que-la-vem-a-historia-1/

**Comment by angellisy** [comment]
A redação ficou a sua cara, Willy :D

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/senta-que-la-vem-a-historia-1/#comment_1087474072>):

> A redação ficou a sua cara, Willy :D

Que bom... eu acho.

**Comment by brunoaugusto** [comment]
![null](<https://i.postimg.cc/RZFsrjJw/e9f64ce38621c75986554abedc96653b.gif>)
